app_builder/lib/app_builder/windows.ex

```diff
@@ -0,0 +1,203 @@
+defmodule AppBuilder.Windows do
+  @moduledoc false
+
+  import AppBuilder.Utils
+  require EEx
+
+  @doc """
+  Creates a Windows installer.
+  """
+  def build_windows_installer(release, options) do
+    tmp_dir = release.path <> "_tmp"
```

Comment:
don't pay too much attention into this file layout. I plan to change this when this lands. In particular, instead of having `$ mix release mac_app` and `$ mix release windows_installer`, I'd like to have a single `$ mix release app` with possible options (e.g. on macOS skip doing .dmg)

Happy to work on this as part of this PR though!

Reply:
A single task sounds great, can definitely be a separate PR!